src/cim/components/Hypershift/DetailsPage/NodePoolsTable.tsx

```diff
@@ -0,0 +1,264 @@
+
+  return (
+    <>
+      <Stack hasGutter>
+        <StackItem>
+          <TableComposable variant="compact">
+            <Thead>
+              <Tr>
+                <Th />
+                <Th>{t('ai:Nodepool')}</Th>
+                <Th>{t('ai:Hostname')}</Th>
+                <Th>{t('ai:Status')}</Th>
+                <Th>{t('ai:Infrastructure env')}</Th>
+                <Th>{t('ai:CPU cores')}</Th>
+                <Th>{t('ai:Memory')}</Th>
+                <Th>{t('ai:Total storage')}</Th>
+                <Th />
+              </Tr>
+            </Thead>
+            <Tbody>
+              {...nodePools.reduce<JSX.Element[]>((rows, np) => {
+                const nodePoolAgents = getNodepoolAgents(np, agents, agentMachines, hostedCluster);
+                const isExpanded = expandedNodePools.some((uid) => uid === np.metadata?.uid);
+
+                const totals = nodePoolAgents.reduce(
+                  (acc, agent) => {
+                    const { memory, cores, disk } = getHostRowHardwareInfo(
+                      agent.status?.inventory || {},
+                    );
+                    acc.memory += +memory.sortableValue;
+                    acc.cores += +cores.sortableValue;
+                    acc.disk += +disk.sortableValue;
+                    return acc;
+                  },
+                  { memory: 0, cores: 0, disk: 0 },
+                );
+
+                rows.push(
+                  <Tr key={np.metadata?.uid}>
+                    <Td
+                      expand={
+                        nodePoolAgents.length
+                          ? {
+                              isExpanded,
+                              rowIndex: 0,
+                              onToggle: () =>
+                                isExpanded
+                                  ? setExpandedNodePools(
+                                      expandedNodePools.filter((uid) => uid !== np.metadata?.uid),
+                                    )
+                                  : setExpandedNodePools([
+                                      ...expandedNodePools,
+                                      np.metadata?.uid || '',
+                                    ]),
+                            }
+                          : undefined
+                      }
+                    />
+                    <Td>{np.metadata?.name}</Td>
+                    <Td>
+                      <Popover
+                        aria-label="node pool hosts"
+                        hasAutoWidth
+                        bodyContent={t('{{agents}} of {{count}} host requested is available', {
+                          agents: nodePoolAgents.length,
+                          count: np.spec.replicas,
+                        })}
+                      >
+                        <Label variant="outline">
+                          {t('{{count}} host requested', { count: np.spec.replicas })}
```

Comment:
Please add 'ai:' prefix to strings thtat have to be translated

Reply:
added